src/pages/community-champions.js

```diff
@@ -370,6 +376,100 @@ const HowToQualifyCard = ({ icon, title, description }) => (
     </HowToQualifyCardContainer>
 );
 
+const Line = styled('hr')`
+    border-color: ${({ theme }) => theme.colorMap.greyDarkOne};
+    margin-bottom: ${LINE_BOTTOM_MARGIN};
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const GreenBulletedListWithGreyText = styled(GreenBulletedList)`
+    grid-area: bullets;
+    margin: ${size.default} 0 0;
+    max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_LIST_MAX_WIDTH};
+    li {
+        margin-bottom: ${size.xsmall};
+    }
+`;
+
+const OtherDetailsAndRequirementsContainer = styled('div')`
+    display: grid;
+    grid-template-areas:
+        'image text'
+        'image bullets';
+    grid-template-columns: ${OTHER_DETAILS_AND_REQUIREMENTS_IMAGE_SIZE} auto;
+    grid-template-rows: auto auto;
+    margin-bottom: ${size.xlarge};
+    @media ${screenSize.upToMedium} {
+        grid-template-areas:
+            'image text'
+            'bullets bullets';
+        margin-bottom: ${size.large};
+    }
+`;
+
+const OtherDetailsAndRequirementsTitle = styled(H5)`
+    @media ${screenSize.upToMedium} {
+        margin-bottom: ${size.tiny};
+    }
+`;
+
+const OtherDetailsAndRequirementsDescription = styled(Description)`
+    max-width: ${OTHER_DETAILS_AND_REQUIREMENTS_DESCRIPTION_MAX_WIDTH};
+`;
+
+const OtherDetailsAndRequirementsImageContainer = styled('div')`
+    align-items: start;
+    display: flex;
+    grid-area: image;
+    justify-content: center;
+`;
+
+const OtherDetailsAndRequirementsImage = styled('img')`
+    width: ${OTHER_DETAILS_AND_REQUIREMENTS_IMAGE_WIDTH};
+`;
+
+const OtherDetailsAndRequirementsBulletText = styled(Description)`
+    margin-bottom: 0;
```

Comment:
With `collapse` do we still need this?

Reply:
ahh i had the same thought. i'm passing this component into the `BulletText` prop of `GreenBulletedList` like `BulletText={OtherDetailsAndRequirementsBulletText}`. i think this can only take in a component name. lol i tried doing `BulletText={<Description collapse/>}` and got an error. however, i can 1) modify `GreenBulletedList` to take in a `bulletTextCollapse` prop and use `collapse` [here](https://github.com/mongodb/devhub/blob/community-champions/src/components/dev-hub/green-bulleted-list.js#L28) or 2) just not use a prop and have it always collapse bc currently all usages of `GreenBulletedList` collapse their `BulletText`

Reply:
Ah we have to pass a component symbol, this would work too:

```
const OtherDetailsAndRequirementsBulletText = ({ children }) => (
    <Description collapse>{children}</Description>
);
```

(We explicitly have to pass `children` for the text)

Reply:
`Description` here should be defining text for a single child (one bullet point) tho, not children. every child is mapped to a list item like this - https://github.com/mongodb/devhub/blob/community-champions/src/components/dev-hub/green-bulleted-list.js#L27-L29

Reply:
Ah okay, you have won me over :)

Reply:
Wait `children` in this case refers to one list item, not the `children` passed to `GreenBulletedListWithGreyText`